app/controllers/boards_controller.rb

```diff
@@ -1,14 +1,14 @@
 class BoardsController < ApplicationController
   def index
-    @boards = Board.includes(:lists)
+    @boards = current_user.ownerships.includes(:lists)
```

Comment:
na to wychodzi ze zalogowany uzytkownik zobaczy tylko boardy w których jest adminem. A co z boardami w których jest memberem ?